src/Xamarin.Android.Build.Tasks/Tasks/CheckGoogleSdkRequirements.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System.IO;
+using System.Linq;
+
+using Java.Interop.Tools.Cecil;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+    public class CheckGoogleSdkRequirments : Task 
+	{
+		[Required]
+		public string TargetFrameworkVersion { get; set; }
+
+		[Required]
+		public string ManifestFile { get; set; }
+
+		public override bool Execute ()
+		{
+			ManifestDocument manifest = new ManifestDocument (ManifestFile, this.Log);
```

Comment:
Do we need to worry about bad XML here? It looks like this would just throw `XmlException`.

Reply:
I don't think so, this takes place after `_CreateBaseApk` and `_GenerateJavaStubs`, if there is a problem with the xml it will have been caught earlier in the build.